sdk/dotnet/Makefile

```diff
@@ -1,18 +1,42 @@
-PROJECT_NAME       := Pulumi .NET Core SDK
-LANGHOST_PKG       := github.com/pulumi/pulumi/sdk/dotnet/cmd/pulumi-language-dotnet
-VERSION            := $(shell ../../scripts/get-version)
-VERSION_DOTNET     := ${VERSION:v%=%}                             # strip v from the beginning
-VERSION_PREFIX     := $(firstword $(subst -, ,${VERSION_DOTNET})) # e.g. 1.5.0
-VERSION_SUFFIX     := $(word 2,$(subst -, ,${VERSION_DOTNET}))    # e.g. alpha.1
-PROJECT_PKGS       := $(shell go list ./cmd...)
-PULUMI_LOCAL_NUGET ?= ~/.nuget/local
+PROJECT_NAME         := Pulumi .NET Core SDK
+LANGHOST_PKG         := github.com/pulumi/pulumi/sdk/dotnet/cmd/pulumi-language-dotnet
+
+PROJECT_PKGS         := $(shell go list ./cmd...)
+PULUMI_LOCAL_NUGET   ?= ~/.nuget/local
+
+VERSION              := $(shell ../../scripts/get-version)
+VERSION_DOTNET       := ${VERSION:v%=%}                                   # strip v from the beginning
+VERSION_FIRST_WORD   := $(strip $(word 1,$(subst -, ,${VERSION_DOTNET}))) # e.g. 1.5.0
+VERSION_SECOND_WORD  := $(strip $(word 2,$(subst -, ,${VERSION_DOTNET}))) # e.g. alpha or alpha.1
+VERSION_THIRD_WORD   := $(strip $(word 3,$(subst -, ,${VERSION_DOTNET}))) # e.g. featbranch or featbranch.1
+
+VERSION_PREFIX       := ${VERSION_FIRST_WORD}
+
+ifeq ($(VERSION_THIRD_WORD),)
+	VERSION_SUFFIX   := ${VERSION_SECOND_WORD}
+else
+	# have to use addsuffix or make adds a whitespace between the two values.
+	VERSION_SUFFIX   := $(addsuffix -$(VERSION_THIRD_WORD),${VERSION_SECOND_WORD})
```

Comment:
NuGet decides on which version is the latest based on alphabetic order:

>Whatever suffixes you use, however, NuGet will give them precedence in reverse alphabetical order

I am not quite sure how `1.5.0-alpha-nugetTest.1572308174+gd139713b ` will stand against `1.5.0-alpha.572308174+gd139713b `

Reply:
i'll delist the nuget-test version.  that was just to validate that branches would work in nuget.  in this case, i think we'll have a good set of of versions since `1.5.0-preview-alpha` will be what we want people to select.